poap/disklayout.erb

```diff
@@ -0,0 +1,5 @@
+<%#
+kind: ptable
+name: NX-OS default fake
```

Comment:
You can specify ```oses``` per https://github.com/theforeman/community-templates#repo-metadata to automatically link it after importing.